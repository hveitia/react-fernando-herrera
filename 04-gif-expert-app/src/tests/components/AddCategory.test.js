import React from 'react';
import { shallow }  from  'enzyme';
import {AddCategory} from '../../components/AddCategory';

describe('Testin on AddCategory component', () => {
    
    const setCategories = () => {};
    const wrapper = shallow(<AddCategory setCategories={setCategories}/>);
    
    test('AddCategory should be render', () => {  
        expect(wrapper).toMatchSnapshot();
    })

    test('Should change the input element.', () => {
       
        const input = wrapper.find('input');
        const value = 'Test';
        input.simulate('change',{ target: {value} });
    })
    
    
})
