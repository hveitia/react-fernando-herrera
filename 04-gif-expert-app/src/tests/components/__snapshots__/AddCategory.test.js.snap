// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testin on AddCategory component AddCategory should be render 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
